Description: disable GUI editor if FCKeditor missing
 only applicable on Debian systems where a system-wide FCKeditor is used
 from a static location and might in some cases be missing
Author: Frank Lin PIAT <fpiat@klabs.be>
Forwarded: not-needed
Last-Update: 2009-11-15
--- a/MoinMoin/config/multiconfig.py
+++ b/MoinMoin/config/multiconfig.py
@@ -1300,6 +1300,14 @@
 _add_options_to_defconfig(options)
 _add_options_to_defconfig(options_no_group_name, False)
 
+# Debian Specific:
+# Try to auto-detect if the package fckeditor is installed
+# (This can be overiden in the wiki configuration file).
+if not os.path.exists('/usr/share/fckeditor/fckeditor.js'):
+     setattr(DefaultConfig, 'editor_force', True)
+     setattr(DefaultConfig, 'editor_default', 'text')
+
+
 # remove the gettext pseudo function
 del _
 
